### PR TITLE
FlutterFire plugin readme updates

### PR DESCRIPTION
**Pull request by mit-mit**
Fixes https://github.com/flutter/flutter/issues/10027

### CODE REVIEW

**Comment by mit-mit** [comment]
@sethladd is https://github.com/flutter/plugins/blob/479e3352f07585d81dc7795cfbd28b3b2261fd4b/FlutterFire.md what you had in mind when you said "I want a FlutterFire overview"?

File: FlutterFire.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# FlutterFire
```
Comment (sethladd) [inline FlutterFire.md]: Let's add a note about what Flutter is. Imagine this page is linked from firebase.google.com.

File: FlutterFire.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+FlutterFire is a set of [Flutter plugins](https://flutter.io/platform-plugins/)
+that enable Flutter Apps to use one or more [Firebase](https://firebase.google.com/) services.
```
Comment (sethladd) [inline FlutterFire.md]: lower-case apps

File: FlutterFire.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Supported Firebase Products
```
Comment (sethladd) [inline FlutterFire.md]: Just want to be sure you want to use the word "Supported" here. We do support them, but Firebase team doesn't support them.

File: FlutterFire.md (hunk 1 above)
Comment (sethladd) [inline FlutterFire.md]: I'd probably also add a note about how official these plugins are. Are these officially from the Firebase team? We can see what they use for wording for their PolymerFire, etc projects.

Comment (mit-mit) [inline FlutterFire.md]: K, how about:

```
FlutterFire is a set of [Flutter plugins](https://flutter.io/platform-plugins/) that enable Flutter apps to use one or more [Firebase](https://firebase.google.com/) services.

[Flutter](https://flutter.io) is a new mobile app SDK to help developers and designers build modern mobile apps for iOS and Android.
```

?

File: FlutterFire.md (hunk 2 above)
Comment (mit-mit) [inline FlutterFire.md]: done

File: FlutterFire.md (hunk 3 above)
Comment (mit-mit) [inline FlutterFire.md]: Changed

File: FlutterFire.md (hunk 1 above)
Comment (mit-mit) [inline FlutterFire.md]: Updated

Comment (mit-mit) [inline FlutterFire.md]: Done

**Comment by mit-mit** [comment]
@sethladd incorporated feedback, can I get a LGTM?

**Comment by sethladd** [comment]
Thank you!

**Pull request merged**